Carting/tests/Application.IntegrationTests/Cart/Commands/AddItemToCartCommandTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿using NUnit.Framework;
+using FluentAssertions;
+using Carting.WebApi.Application.Common.Exceptions;
+using Carting.WebApi.Application.Carts.Commands.AddItemToCart;
+
+namespace Carting.Application.IntegrationTests.Cart.Commands;
+
+using static Testing;
+
+public class AddItemToCartCommandTests : BaseTestFixture
+{
+    [Test]
+    public async Task ShouldCreateCart()
```

Comment:
With my current design, it's not possible. I've been thinking about if having an empty cart would make sense. 

Cart doesn't have any information apart from the items and its unique identifier. Should we attach additional properties, it will definitely make sense to change this logic, I believe.

Reply:
My suggestion would be make it more consistent.
* If you cant create empty cart, so after removing last item from it delete it as well.
* Or allow creation of empty cart

If you leave empty cart it has to be associated with something whether it be session or user or something else in order to be reused